.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+package-lock.json
```

Comment:
Why did you add the package lock to the ignore file?

Without it, everyone installing dependencies might get different versions to you or anyone else.